sql/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -313,4 +314,21 @@ object Connect {
       .internal()
       .booleanConf
       .createWithDefault(true)
+
+  val CONNECT_AUTHENTICATE_TOKEN =
+    buildStaticConf("spark.connect.authenticate.token")
```

Comment:
Nit: Should we use the verb `Authenticate` or the noun `Authentication`? I'd lean towards the noun, however I am an ESL speaker.

Reply:
We use Authenticate everywhere else. Let's be consistent :)...

Reply:
Yeah I wasn't sure, just modeled after `spark.authenticate`, which itself makes sense as a verb, but then there's `spark.authenticate.secret`. Initially I had this as `spark.connect.authenticate.secret`, but then changed it to `token`. I don't have a strong preference. Could go back to `spark.connect.authenticate.secret` for consistency, or change to `spark.connect.authentication.token`